lib/sedna/core/lifelong_learning/lifelong_learning.py

```diff
@@ -34,40 +33,39 @@ class LifelongLearning(JobBase):
 
     def __init__(self,
```

Comment:
Quite a lot contents in the __init__ function (100+ loc), without any comments, making it hard to read. It would be nice to add comments time-to-time in the middle.

Reply:
Yep

Reply:
So will that be tackled in the future? It would be great if we have related issues in the community.